### ISSUE TITLE
2021 [CVPR] DatasetGAN: Efficient Labeled Data Factory with Minimal Human Effort

### ISSUE THREAD

**Issue by pomelyu** [pomelyu/paper-reading-notes#10]
## Introduction
<img width="1014" alt="image" src="https://github.com/pomelyu/paper-reading-notes/assets/2679561/a9965c09-8945-452e-a113-4f6ff4534bc6">

## Method
- extract the feature layers from pretrained StyleGAN
- upsample these feature layers to fixed size and concatenate them
- train a per-pixel MLP class classifier to map these features to ground-truth segmentation map

## Highlight
- require only few data for training and hence can use the detailed labels

## Limitation
- require large GPU memory(160G) for training
